### DISCUSSION TITLE
Correctly activating 12 Step Meeting List Plugin

### DISCUSSION THREAD

**Discussion by DanMTampa** [code4recovery/12-step-meeting-list#1153] [Q&A]
I re-constructed our Intergroup Website and need to be sure I have the meeting plugin properly activated for our area. 
New site is in Wordpress with Twenty Twenty Three theme. 

I imported the data from the old site (still active) and the list is functioning. 
I am not sure what steps are next to replace one site with the new one or if multiple can be active at the same time. 

Site address: https://e7u.a23.myftpupload.com/
Meeting List: https://e7u.a23.myftpupload.com/meetings/

Dan Meeks

**Comment by joshreisner** [comment]
HI Dan,

So you're about to change the URL of your demo site to be aatampa-area.org and then take down the website at meetings.aatampa-area.org? 

Once you change the URL of the site, I would suggest three things:
- On the new site, in WordPress Settings, make sure the URLs there are the live URL: https://aatampa-area.org/
- Go to WordPress Settings > Permalinks > Save Changes (this often helps)
- Go to any meeting and hit Update

At this point your new site should be working properly. If you are connected to the Meeting Guide app your feed URL has now changed, so I would let them know the new one and that they need to update it now on their end on their end: https://meetingguide.helpdocs.io/article/g0ykqkdq0u-connecting-to-meeting-guide-step-by-step

Finally, take down the old meeting finder site. It would be good to leave a message up for a period of time that redirects users from https://www.meetings.aatampa-area.org/meetings/ to https://aatampa-area.org/meetings/ because people will have bookmarked the meeting list, and it will also take Google some time to update. One option if you want to transparently redirect users is to uninstall TSML from that site and use a plugin like https://wordpress.org/plugins/eps-301-redirects/ to redirect users transparently

Hope that helps

**Reply by DanMTampa** [reply]
Sorry for the confusion. 
My UN: DanMTampa is what I use on most all social media and discussion forums since the late 90s.

I have since retired to Citrus County / District 28 / Area 14 / Nature Coast Intergroup. 

It looks like most of your response still applies. 

I am interested in switching to the TSML UI. 
Is that covered in a document?

**Reply by joshreisner** [reply]
Oh great. You should be able to change to TSML UI by going to WordPress > Meetings > Settings and changing the appearance dropdown
